lib/bootloader_setup.pm

```diff
@@ -35,17 +36,33 @@ our @EXPORT = qw(
   tianocore_select_bootloader
 );
 
+# prevent grub2 timeout; 'esc' would be cleaner, but grub2-efi falls to the menu then
+# 'up' also works in textmode and UEFI menues.
+sub stop_grub_timeout {
+    send_key 'up';
```

Comment:
yes ,I was thinking about the future, when it will be used on another places like on ppc64le bootmenu it can be problem, as the menu is oppositely oriented as all other architectures
now it's ok